doc/convertibles.rdoc

```diff
@@ -0,0 +1,187 @@
+== Class-Convertible Objects
```

Comment:
Did you find this terminology somewhere else, or did you create it?

Reply:
As far as I know, there's no existing discussion of this anywhere.  So maybe a new term.

Reply:
Okay, I will double check this.

Great to introduce some concept to explain it, but maybe there is existing terminology. It's 2AM but I will check tomorrow.